Gaijinizer patch/Map465.txt

```diff
@@ -3,493 +3,634 @@
-
+> CONTEXT: Map465/events/3/pages/0/82/Dialogue/sirousagi_fc1
+But... before all that...
+Since you've got yourself wrapped up in all this
+confusion, Mizuki-chan, I'd better fill you in on
+what's going on.#Ron
 > END STRING
 
 > BEGIN STRING
 自分の世界がああなっちゃった理由も説明しないとだし
 
 またいつ慌ただしくなるかも分からないから………
-> CONTEXT: Map465/events/3/pages/0/87/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/87/Dialogue/sirousagi_fc1
+We don't know when things will pop off again, so
+I'll also explain the reason why your world ended
+up like it did back there.#RonLocalization
 > END STRING
 
 > BEGIN STRING
 …………正直、面子に夜深ちゃんが含まれていなかったら
 
 あんな得体の知れない亀裂になんか入らなかったわ……
-> CONTEXT: Map465/events/3/pages/0/94/Dialogue/enemy38 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/94/Dialogue/enemy38
+To be honest, if Yami-chan hadn't been part of
+your group, I wouldn't have jumped into such a
+weird rift.#Ron
 > END STRING
 
 > BEGIN STRING
 説明してもらおうかしら……
 
 私の世界で、一体何が起こったのかを。
-> CONTEXT: Map465/events/3/pages/0/98/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map465/events/3/pages/0/98/Dialogue/enemy38
+I guess I'll get you to explain it all.
 
+Just what on earth happened to my world?#Ron
 > END STRING
 
 > BEGIN STRING
 ………アバドーンによる世界の崩壊、ねぇ……
 
 そして、それを阻止するための『ゲスト』が私…？
 ついでにアンタが、道案内をする『白兎』……
-> CONTEXT: Map465/events/3/pages/0/112/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map465/events/3/pages/0/112/Dialogue/enemy38
+So this Abaddon is causing worlds to collapse...
+And I'm a "guest" that's meant to prevent that...
 
+And you, White Rabbit, happen to be our guide?
 > END STRING
 
 > BEGIN STRING
 トドメに、『アリス』に『チェシャ』に『お茶会』……
 
 やれやれ……鏡の世界にでも迷い込んだのかしら……？
-> CONTEXT: Map465/events/3/pages/0/117/Dialogue/enemy38 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/117/Dialogue/enemy38
+And to top it off, there's Alice, Cheshire,
+and some tea party? Good grief, have I stumbled
+through the looking-glass?#Ron
 > END STRING
 
 > BEGIN STRING
 いや……あの童話は、悪魔や魔界の伝承にも
 知らぬうちに入り込んでいる、謎多き言い伝えだ……
 
 単語が一致したとしても、さして不思議ではない。
-> CONTEXT: Map465/events/3/pages/0/121/Dialogue/enemy37 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/121/Dialogue/enemy37
+Well, that fairy tale is an enigmatic story that
+quickly made its way into the folklore of demons
+and the Demon Realm.#RonIffy
 > END STRING
 
 > BEGIN STRING
 それよりも問題は『アバドーン』の実体化……
 
 確かにあの虫食いがヤツの仕業なら説明がつくが……
 何故今になって実体化を……？
-> CONTEXT: Map465/events/3/pages/0/126/Dialogue/enemy37 < UNTRANSLATED
-
+> CONTEXT: Map465/events/3/pages/0/126/Dialogue/enemy37
+But putting that aside, the problem is Abaddon's
+manifestation. If those holes in our world are
+his doing, then it makes sense, but why is he
+manifesting now?#Ron
 > END STRING
 
 > BEGIN STRING
 アイツ、タダノ魔界ノ『ブラックホール』
 
 食ベルダケ。悪巧ミ、シナイ
-> CONTEXT: Map465/events/3/pages/0/131/Dialogue/enemy37 < UNTRANSLATED
+> CONTEXT: Map465/events/3/pages/0/131/Dialogue/enemy37
+He simply Demon Realm's "Black Hole"...
 
+He just eats, no plan.#RonIffy
 > END STRING
```

Comment:
Rewording for flow:
```
He's simply like a demonic "black hole"...

He consumes everything without thinking.
```

(Looks like the character who's speaking is Orthrus, so you'll have to transcribe it into his style better as well as this.)